### ISSUE TITLE
Pedlar buys on last tick rather than next tick. 

### ISSUE THREAD

**Issue by Joearrowsmith** [nuric/pedlar#2]
This enables an order to be closed outside of a market hours in both live trading and backtesting. It is also a type of time leakage.

**Comment by nuric** [comment]
This is true in that it doesn't give a true market value and the orders can be closed on the weekend. There would need to be a server side queue to close orders and on next tick it would close them. To implement I reckon we would need:

 - A server side queue of orders, in the most basic case a Python list of order ids.
 - On new tick, the server checks if there are any orders to be closed.
 - Closes the orders based on the latest tick.

These changes would queue the order during the weekend and on Monday when the first tick arrives would close the orders.
